fearless-utils/src/testShared/java/jp/co/soramitsu/fearless_utils/encrypt/mnemonic/SubstrateSeedFactoryTest.kt

```diff
@@ -0,0 +1,49 @@
+package jp.co.soramitsu.fearless_utils.encrypt.mnemonic
+
+import com.google.gson.Gson
+import jp.co.soramitsu.fearless_utils.encrypt.EncryptionType
+import jp.co.soramitsu.fearless_utils.encrypt.keypair.substrate.SubstrateKeypairFactory
+import jp.co.soramitsu.fearless_utils.extensions.toHexString
+import jp.co.soramitsu.fearless_utils.getResourceReader
+import jp.co.soramitsu.fearless_utils.encrypt.junction.SubstrateJunctionDecoder
+import jp.co.soramitsu.fearless_utils.encrypt.seed.substrate.SubstrateSeedFactory
+import org.junit.Assert
+
+abstract class SubstrateSeedFactoryTest {
```

Comment:
I suggest to rename it as well to a `SubstrateKeypairGenerationTest`or something like this

Reply:
Done